Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/azpy/dcc/substance/sbs_export.py

```diff
@@ -0,0 +1,10 @@
+from pysbs import context
```

Comment:
consider a better API structure that can grow over time ... maybe even stub out a scaffold that covers sbs and sbsar?
maybe ...
import azpy.dcc.substance.sbs as dccsi_sbs
dccsi_sbs.read()
dccsi_sbs.export()

Over time if substance\sbs.py gets large, it could be turned into a package of multiple .py files:

azpy\dcc\substance\sbs\_init_.py
azpy\dcc\substance\sbs\read.py
azpy\dcc\substance\sbs\export.py